examples/audio/eeg_signal.py

```diff
@@ -0,0 +1,527 @@
+                "thinkOfItems-ver2": "thinkOfItems",
+                "video-ver1": "video",
+                "video-ver2": "video",
+                "thinkOfItemsInstruction-ver1": "thinkOfItemsInstruction",
+                "thinkOfItemsInstruction-ver2": "thinkOfItemsInstruction",
+                "colorRound1-1": "colorRound1",
+                "colorRound1-2": "colorRound1",
+                "colorRound1-3": "colorRound1",
+                "colorRound1-4": "colorRound1",
+                "colorRound1-5": "colorRound1",
+                "colorRound1-6": "colorRound1",
+                "colorRound2-1": "colorRound2",
+                "colorRound2-2": "colorRound2",
+                "colorRound2-3": "colorRound2",
+                "colorRound2-4": "colorRound2",
+                "colorRound2-5": "colorRound2",
+                "colorRound2-6": "colorRound2",
+                "colorRound3-1": "colorRound3",
+                "colorRound3-2": "colorRound3",
+                "colorRound3-3": "colorRound3",
+                "colorRound3-4": "colorRound3",
+                "colorRound3-5": "colorRound3",
+                "colorRound3-6": "colorRound3",
+                "colorRound4-1": "colorRound4",
+                "colorRound4-2": "colorRound4",
+                "colorRound4-3": "colorRound4",
+                "colorRound4-4": "colorRound4",
+                "colorRound4-5": "colorRound4",
+                "colorRound4-6": "colorRound4",
+                "colorRound5-1": "colorRound5",
+                "colorRound5-2": "colorRound5",
+                "colorRound5-3": "colorRound5",
+                "colorRound5-4": "colorRound5",
+                "colorRound5-5": "colorRound5",
+                "colorRound5-6": "colorRound5",
+                "colorInstruction1": "colorInstruction",
+                "colorInstruction2": "colorInstruction",
+                "readyRound1": "readyRound",
+                "readyRound2": "readyRound",
+                "readyRound3": "readyRound",
+                "readyRound4": "readyRound",
+                "readyRound5": "readyRound",
+                "colorRound1": "colorRound",
+                "colorRound2": "colorRound",
+                "colorRound3": "colorRound",
+                "colorRound4": "colorRound",
+                "colorRound5": "colorRound",
+            }
+        },
+        inplace=True,
+    )
+
+
+updateLabel(eeg)
+
+print("After replacing labels")
+print(eeg["label"].unique())
+
+le = LabelEncoder()
+le.fit(eeg["label"])
+eeg["label"] = le.transform(eeg["label"])
+
+"""
+We extract the number of unique classes present in the data
+"""
+
+num_classes = len(eeg["label"].unique())
+print(num_classes)
+
+"""
+We now visualize the number of samples present in each class using a Bar plot.
+"""
+
+plt.bar(range(num_classes), eeg["label"].value_counts())
+plt.title("Number of samples per class")
+plt.show()
+
+"""
+## Scale data, prepare `tf.data.Dataset` and make Class Weights using Naive methods
```

Comment:
this header is a bit noisy, do some of these tasks deserve subsections?  maybe move the tf.data.dataset piece to its own section

Reply:
Will do. I'll break the Scaling, preparation and class-weight prep into separate sections